geoips/schema/products/bases/product.yaml

```diff
@@ -16,9 +16,9 @@ properties:
 
   spec:
     type: object
-    required:
-      - source_name
-      - variables
+    # required:
```

Comment:
These are temporarily commented out to allow data_fusion products to validate?

Reply:
Yes, exactly right.